witness/README.md

```diff
@@ -0,0 +1,44 @@
+Witnessing
+============
+
+Witnessing is an important part of any transparent ecosystem, as it ensures that
+misbehavior on the part of logs can be prevented or detected. This is done by
+ensuring that the _checkpoints_ published by logs are _consistent_, which means 
+the log is presenting a globally consistent view of its contents to everyone and
+thus isn't carrying out a _split-view attack_.
+
+In terms of checkpoints, we consider witness implementations that use the
```

Comment:
This sentence leaves my brain hanging, pending more input. We consider foos that use bar... more words needed?!